indico/legacy/webinterface/tpls/RoomBookingNewBookingInfoWidget.tpl

```diff
@@ -1,44 +1,52 @@
 <%page args="form=None"/>
 
 <%
-    fields = ['booked_for_user', 'contact_email', 'contact_phone', 'booking_reason']
+    fields = ['booked_for_user', 'booking_reason']
     field_args = {
         'booking_reason': {'rows': 3, 'placeholder': _(u'Reason...')}
     }
     helpers = {
-        'contact_email': 'You can specify multiple email addresses separated by commas.',
-        'booking_reason': 'Specify why you need this room.'
+        'booking_reason': 'Specify why you need this room.',
+        'booked_for_user': 'Specify who will be using the room.'
+    }
+    ids = {
+        'booked_for_user': 'booked-for-user-wrapper',
+        'booking_reason': 'booking-reason'
     }
 %>
 
 <div id="bookedForInfo">
+    <div id="other-user">
+        <input type="radio" name="other_user" value="False">${ _("I'll be using the room") }<br>
+        <input type="radio" name="other_user" value="True">${ _("I'm making the booking for someone else") }<br>
+    </div>
     % for field in fields:
-        <div class="toolbar thin space-before space-after">
+        <div id="${ ids[field] }" class="toolbar thin space-before space-after">
```

Comment:
Did you mean `form[field].id` ? Or that i should make `fields` a dictionary with names and ids ? I need different id for the div and different for input element in booking-for-user